components/ui/NavBar.tsx

```diff
@@ -13,9 +30,9 @@ export const Navbar = ({ items }: { items: NavItem[] }) => {
         >
           <h1 className="text-xl md:text-3xl">{siteConfig.name}</h1>
         </Link>
+        <ul className="flex text-xl md:text-2xl">
```

Comment:
The `<ul>` tag can be a `<nav>`, and then we don't need the `<li>` tags. A cleaner structure.